addons/edi/models/edi_document.py

```diff
@@ -124,7 +141,7 @@ def autocreate(self, inputs, allow_process=True):
             doc_create_data = {
                 "doc_type_id": autodetect.type.id,
             }
-            if allow_process:
+            if allow_process and self.processing_level in ("prepare", "execute"):
```

Comment:
Is allow_process field used for other purposes, i thing a single field can handle what we want

Reply:
Allow process can come from the gateway, in case we want to generally disable the processing of files at the GW level instead of EDI document level. I think it is fine as it is. Thanks for the suggestion.